spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2220,6 +2220,16 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
               expr.children(2))
             None
           }
+        case expr if expr.prettyName == "size" =>
```

Comment:
for some reason, it's size and not array_size

Reply:
`ArraySize` is `RuntimeReplaceable` in Spark so it is replaced by the optimizer to `Size` at runtime.